test/studies/champs/sub_test

```diff
@@ -43,53 +67,64 @@ if [ "${CHPL_LLVM}" != "none" ] ; then
   apply_patch $CHAMPS_PATCH_PATH/optional/llvm.patch
 fi
 
-# copy the prepartitoned grid for 16 locales into scratch space for better IO
-# performance
-cp $CHAMPS_DATA_PATH/CRMHL_coarse_1152B.cgns $CHAMPS_HOME/
-
-# in the system where we run this, compute nodes don't have access to our shared
-# non-scratch space. I do not want to store those in the scratch space for good.
-# So, what we are doing here is to copy the stuff from CHAMPS_CFG_PATH and
-# CHAMPS_DEP_PATH to the scratch space for running the code.
-cp $CHAMPS_CFG_PATH/flow.in $CHAMPS_HOME/
-
-# ... and this is where it really gets unfortunate.
-cp -r $CHAMPS_DEP_PATH $CHAMPS_HOME/
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CHAMPS_HOME/deps-manual/lib
+if [ ! -z "$CHAMPS_QUICKSTART" ]; then
+  echo "[Patch commit created]"
+  git commit -am "Snapshot after applying patches" >/dev/null
+  git clean -df >/dev/null
+  git status
+fi
 
 # some debugging output before starting the run
 echo "contents of $CHAMPS_HOME"
 ls $CHAMPS_HOME
-echo LD_LIBRARY_PATH=$LD_LIBRARY_PATH
 
 # Compile CHAMPS executables
-test_compile prep
-test_compile flow
 test_compile icing
-test_compile drop
-test_compile potential
-test_compile potentialPrep
-test_compile geo
-test_compile thermo
-test_compile postLink
-test_compile post
-test_compile stochasticIcing
-test_compile octree
-test_compile eigenSolvePost
-test_compile externalSolverPost
-test_compile meshDeformation
-test_compile continuation
-test_compile stability
-test_compile coloring
-
-# copy the input data to the filesystem we're running on
-
-# I can't tell whether prep is running too slow or just freezing, for now don't run it
-# cp $CHAMPS_DATA_PATH/CRMHL_coarse.cgns $CHAMPS_HOME/
-# test_run prep 1
-
-test_run flow 16
-
-$CHPL_HOME/util/test/genGraphs --configs $CHPL_TEST_PERF_CONFIGS --annotate $CHPL_HOME/test/ANNOTATIONS.yaml --name $CHPL_TEST_PERF_CONFIG_NAME --perfdir $CHPL_TEST_PERF_DIR --outdir $CHPL_TEST_PERF_DIR/html --graphlist $CHAMPS_GRAPH_PATH/GRAPHLIST --testdir $CHAMPS_GRAPH_PATH --alttitle "CHAMPS Performance Graphs"
+
+if [ -z "$CHAMPS_QUICKSTART" ] || [ ! -z "$CHAMPS_COMPILE_ALL_EXECS" ] ; then
+  test_compile prep
+  test_compile flow
+  test_compile drop
+  test_compile potential
+  test_compile potentialPrep
+  test_compile geo
+  test_compile thermo
+  test_compile postLink
+  test_compile post
+  test_compile stochasticIcing
+  test_compile octree
+  test_compile eigenSolvePost
+  test_compile externalSolverPost
+  test_compile meshDeformation
+  test_compile continuation
+  test_compile stability
+  test_compile coloring
+fi
+
+
+if [ -z "$CHAMPS_QUICKSTART" ] ; then
+  # copy the input data to the filesystem we're running on
+  # I can't tell whether prep is running too slow or just freezing, for now don't run it
+  # cp $CHAMPS_DATA_PATH/CRMHL_coarse.cgns $CHAMPS_HOME/
```

Comment:
I'm assuming these two lines of code were commented out on purpose as the "running prep is too slow so we'll copy it instead on line 111) thing".

Not asking for any change here, just pointing it out in case my assumption is wrong.

Reply:
You're not wrong, and this is not new: https://github.com/chapel-lang/chapel/pull/23444/files/f63665ff36777ba4a4c590543c4a31df6c191423#diff-458d1e86eec8ab02b56057df6fb49ed344032c25552dd5813c4b02f652a5898cL87-L89